routes/chat.js

```diff
@@ -107,20 +68,56 @@ router.post('/chat', async function (req, res) {
       is_group: false,
       messages: [],
     });
-    const chatID = chat._id;
+    const chatId = chat._id;
 
     await UserChat.create({
       user: user._id,
-      chat: chatID,
+      chat: chatId,
     });
     await UserChat.create({
       user: otherUser._id,
-      chat: chatID,
+      chat: chatId,
     });
 
-    res.json(chat);
+    res.status(201).json(chat);
   } else {
     // join
-    res.status(204).json(commonChats);
+    res.status(200).json(commonChats);
   }
 });
+
+router.post('/chat/:chatId/messages', async function (req, res) {
+  if (!ObjectId.isValid(req.params.chatId)) {
```

Comment:
put chatId in a const variable to check it, otherwise when finding it elsewhere we don't know if it's already been checked.